contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -33,13 +33,18 @@
                  Datetime of the start of the measurement.
             </doc>
         </field>
+        <field name="end_time" type="NX_DATE_TIME">
+            <doc>
+                 Datetime of the end of the measurement.
+            </doc>
+        </field>
         <field name="definition">
             <attribute name="version"/>
             <enumeration>
                 <item value="NXmpes"/>
             </enumeration>
         </field>
-        <group type="NXuser">
+        <group type="NXuser" recommended="true">
```

Comment:
This is open for discussion and it is a general question how requiring a name and other identifying information is in accordance with GDPR or other data privacy regulations.

Reply:
This is delicate. Other FAIRification attempts have created permanent identifiers for the users and recorded only that. This means that all information about the author are not readable to anyone, but they are there, accessible only through interpretation. 
I personally think that authorship should be public and human readable.
Maybe it could be interesting to use both approaches: some information (name, affiliation) about the author is public (of course the author then has to tick a consent box), and other (email, phone number etc.) are optional and attached to the file only via permanent identifiers, putting them behind a privacy screen.

Reply:
> This is delicate. Other FAIRification attempts have created permanent identifiers for the users and recorded only that. This means that all information about the author are not readable to anyone, but they are there, accessible only through interpretation. 

We discussed this at some point as well. While I like the idea it ties the format, however, to a third party service to resolve the permanent id (like, e.g., orchid).

> I personally think that authorship should be public and human readable. 

I agree as well. If we say data should follow a similar publishing schema to papers it *should* also demand authorship information.

> Maybe it could be interesting to use both approaches: some information (name, affiliation) about the author is public (of course the author then has to tick a consent box), and other (email, phone number etc.) are optional and attached to the file only via permanent identifiers, putting them behind a privacy screen.

This could be a use for abstract base classes as well. We could have different classes you could provide: Name + Email, Name + Orchid, Only orchid, another third party identifying service.
However, having a consent box is tricky as we do not have any lever on how the data gets created, we just write down which data we expect or require.

Reply:
There are also the nfdi base services adding support for identification information: https://base4nfdi.de/component/content/article/page-2?catid=2&Itemid=342

Reply:
I have created a separate issue to keep track of these different ideas (#138). Feel free to resolve here.

Reply:
As for the refactoring of EM and APM, I worked the following way: the existence constraint on group instances of NXuser are set to optional. If used, at least setting name is recommended. Personally, I am convinced that having an instance of NXidentifier in NXuser suffices. If that is then needed, NXidentifier can be extended with adding the option for this representing the concept of a decentralized identifier. For a given installation of an OASIS indeed it is the responsibility of the specific group of people who organize internally how they manage an OASIS in that this OASIS is configured in a way to not incentivize users or stricter to not allow people to breach the GDPR. Not our responsibility but indeed setting NXuser and name required right from the start is questionable as we encourage if not enforce problems. The real question though is another one: If I document in NXuser that there is a concept with symbol "name" and I write my name there what triggers that there is now the connection with me as a legal person. The string of my name is not unique. With NXidentifiers that is more tricky.
In short, technically I think this is a solved problem, NFDI decides as a whole we make a few fixes in our machinery that complies to this.
On another note, if I add a DOI of a publication to an appdef and define the concept of that DOI such like "paper whose first author is the person who performed the measurement and is legally responsible" will this have any legal value in court?  Indeed, fascinating questions, beyond our authority to decide.

Reply:
I will resolve here. @mkuehbach I have added your comment to the new issue.